src/test/java/com/gooddata/collections/UriPageTest.java

```diff
@@ -38,6 +38,8 @@ public void test() throws Exception {
         final UriComponentsBuilder builder = fromUriString("/this/is/{template}").query("other=false");
 
         uri.updateWithPageParams(builder);
+        uri.updateWithPageParams(builder);
```

Comment:
oh, i see. can you please make a bit explicit - create a new test method with a descriptive name